src/main/java/com/lorepo/icplayer/client/addonsLoader/LocalAddonsLoader.java

```diff
@@ -44,14 +46,22 @@ public LocalAddonsLoader() {}
 
 	@Override
 	public void load(ILoadListener callbacks) {
-		if (this.requestSend) {
-			if(this.addonsXMLFetched) {
-				this.flushAddon(this.currentAddonDescriptor, callbacks);
+		if (this.firstAddonRequestSent) {
+			if (this.firstAddonXMLFetched) {
+				if (this.requestSend) {
```

Comment:
Znaczenie tej zmiennej jak i metody requestLoad się zmieniło w kontekście do nowej całości. Z mojego punktu widzenia ciężko się to czyta nie przez if-y (choć trochę utrudniają) a przez używanie starego nazewnictwa. Jeżeli dobrze rozumiem to obecnie:
* requestSend - true jeżeli został wysłany request na załadowanie addons.min.xml
* requestLoad - wyślij request o addons.min.xml
Teraz, gdy jest więcej możliwości wysłania requestu to te proste nazwy (dobre z perspektywy starej implementacji) wprowadzają utrudnienie w czytaniu.